package.json

```diff
@@ -53,7 +53,7 @@
     "test": "jest --config jest/unit.js",
     "test-ci": "jest --config jest/unit-ci.js --ci -w 1",
     "e2e": "jest --config jest/e2e.js",
-    "e2e-ci": "jest --config jest/e2e-ci.js --ci -w 1",
+    "e2e-ci": "node --no-warnings ./node_modules/.bin/jest --config jest/e2e-ci.js --ci -w 1",
```

Comment:
We should leave warnings on and instead eliminate the warnings everywhere.

Reply:
You can ignore the no-warnings I was just seeing if it changed something with CI.

All `import * as <> from ‘crypto’` are out of the code base.